assets/src/edit-story/components/helpCenter/quickTip/index.js

```diff
@@ -62,16 +62,22 @@ export function QuickTip({
   title,
   description,
   isLeftToRightTransition = true,
+  figure,
   ...transitionProps
 }) {
+  const { assetsURL } = useConfig();
   return (
     <Transitioner
       {...transitionProps}
       isLeftToRightTransition={isLeftToRightTransition}
     >
       <Panel>
         <Overflow>
-          <Figure />
+          {Boolean(figure) && (
+            <Figure controls={false} autoPlay loop muted>
```

Comment:
Probably want to add `noControls` prop as well. Doesn't make sense to show controls for this kind  of animation.

```suggestion
            <Figure controls={false} autoPlay loop muted noControls>
```